docs/guide/output-data-widgets.md

```diff
@@ -345,8 +345,8 @@ For filtering data, the GridView needs a [model](structure-models.md) that repre
 usually taken from the filter fields in the GridView table.
 A common practice when using [active records](db-active-record.md) is to create a search Model class
 that provides needed functionality (it can be generated for you by [Gii](start-gii.md)). This class defines the validation
-rules for the search and provides a `search()` method that will return the data provider with an
-adjusted query that respects the search criteria.
+rules to show filter controls on the GridView table and to provides a `search()` method that will return the data 
+provider with an adjusted query that respects the search criteria.
```

Comment:
Usage of "an" in phrase "an adjusted query" is correct. Not sure if it's needed at all, we can write "with adjusted query" in this sentence.